eldritch/ancient_ones.py

```diff
@@ -0,0 +1,147 @@
+import abc
+
+from eldritch import events, places
+from eldritch.monsters import Monster
+
+
+class AncientOne(metclass=abc.ABCMeta):
+  def __init__(self):
+    self.doom = 0
+    self.combat_difficulty = 0
+    self.attributes = []
+
+  def get_interrupts(self, owner, state):
+    return []
+
+  def get_modifier(self, thing, attribute):
+    return 0
+
+  def awaken(self, state):
+    return []
+
+  @abc.abstractmethod
+  def attack(self, state):
+    raise NotImplementedError
+
+
+class SquidFace(AncientOne):
+  def __init__(self):
+    super(AncientOne, self).__init__()
+    self.name = "SquidFace"
+    self.max_doom = 13
+    self.combat_difficulty = -6
+
+  def get_modifier(self, thing, attribute):
+    if isinstance(thing, Monster) and thing.name == "Cultist":
+      return {'horror': -2, 'horror_damage': 2}.get(attribute, 0)
+    if isinstance(thing, characters.Character):
+      return {'maximum_sanity': -1, 'maximum_stamina': -1}
+    return 0
+
+  def attack(self, state):
+    self.doom = min(self.doom + 1, self.max_doom)
+    # TODO: Each character lowers max sanity or stamina
+
+
+class YellowKing(AncientOne):
+  def __init__(self):
+    super(AncientOne, self).__init__()
+    self.name = "The Yellow King"
+    self.attributes = ['physical resistance']
+    self.combat_difficulty = None
+    self.luck_modifier = 1
+
+  def awaken(self, state):
+    self.combat_difficulty = state.terror
+
+  def get_modifier(self, thing, attribute):
+    if isinstance(thing, Monster) and thing.name == "Cultist":
+      return {'combat_difficulty': -2, 'movement': 'flying'}.get(attribute, 0)
+    if isinstance(thing, events.GateCloseAttempt):
+      return 3
+    return 0
+
+  def attack(self, state):
+    for char in state.characters:
+      check = events.Check(char, "luck", self.luck_modifier)
+      state.event_stack.append(
```

Comment:
You should be appending a sequence of pass/fails, not appending all of them on top of eachother.

Reply:
Done.